pkgs/tools/security/echidna/default.nix

```diff
@@ -1,53 +1,107 @@
-{ lib
-, mkDerivation
-, fetchFromGitHub
-, haskellPackages
-, slither-analyzer
+{
+  lib,
+  fetchpatch,
+  mkDerivation,
+  fetchFromGitHub,
+  haskellPackages,
+  slither-analyzer,
 }:
 
 mkDerivation rec {
   pname = "echidna";
-  version = "2.2.2";
+  version = "2.2.3";
 
   src = fetchFromGitHub {
     owner = "crytic";
     repo = "echidna";
     rev = "v${version}";
-    sha256 = "sha256-l1ILdO+xb0zx/TFM6Am9j5hq1RnIMNf2HU6YvslAj0w=";
+    sha256 = "sha256-NJ2G6EkexYE4P3GD7PZ+lLEs1dqnoqIB2zfAOD5SQ8M=";
   };
 
-  isLibrary = true;
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/crytic/echidna/compare/v2.2.3...hellwolf:echidna:2913b02.patch";
```

Comment:
```suggestion
      url = "https://github.com/crytic/echidna/commit/2913b027d7e793390ed489ef6a47d23ec9b3c800.patch";
```

Reply:
Okay, sure. I thought the tag was "immutable" enough.